testdata/v3/simple/web/handler.go

```diff
@@ -132,3 +132,8 @@ type Fish struct {
 	Breed string `json:"breed"`
 	Size  int    `json:"size"`
 }
+
+type CatchedFishForm struct {
+	Count uint  `form:"count"`
+	Fish  *Fish `form:"-"`
+}
```

Comment:
So if you apply this changes of `testdata/v3/simple/api/api.go` and `testdata/v3/simple/web/handler.go` to the branch `v2` and try to generate docs

```
swag init --v3.1
```

It'll panic